src/runtime/description.ts

```diff
@@ -71,11 +71,7 @@ export class Description {
 
   getRecipeSuggestion(formatterClass = DescriptionFormatter) {
     const formatter = new (formatterClass)(this.particleDescriptions, this.storeDescById);
-    const desc = formatter.getDescription(this.arcRecipes[this.arcRecipes.length - 1]);
-    if (desc) {
-      return desc;
-    }
-    return formatter._capitalizeAndPunctuate(this.arcRecipeName || Description.defaultDescription);
+    return formatter.getDescription(this.arcRecipes[this.arcRecipes.length - 1]);
```

Comment:
could you, please, remove `defaultDescription ` and `arcRecipeName` altogether? it is no longer used.

Reply:
Done.